proto/stride/stakeibc/trade_route.proto

```diff
@@ -39,15 +41,32 @@ message TradeRoute {
   // Currently Osmosis is the only trade chain so this is an osmosis pool id
   uint64 pool_id = 9;
 
-  // Spot price is a decimal ratio of the input to output denom as a string
-  string spot_price = 10;
+  // price of the output denom relative to the input denom
+  // this is update periodically via an ICQ
+  string host_token_price = 10 [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.nullable) = false
+  ];
+  // timestamp that the price was last updated
+  google.protobuf.Timestamp last_price_update_time = 11
```

Comment:
last is implied, since there are many time related types what about price_update_timestamp as a name?

Reply:
i'm good with that!